src/Native/Unix/Common/pal_utilities.h

```diff
@@ -16,6 +16,19 @@
 #include <type_traits>
 #include <unistd.h>
 
+#ifdef DEBUG
+#define assert_msg(cond, msg, errno) do \
```

Comment:
Nit: maybe a different variable name than `errno`, which could be confusing since it's already defined?